linkerd/core/src/main/scala/io/buoyant/linkerd/Linker.scala

```diff
@@ -104,24 +105,33 @@ object Linker {
     admin: Option[AdminConfig]
   ) {
 
-    def mk(defaultTelemeter: Telemeter = NullTelemeter): Linker = {
+    def mk(): Linker = {
       // At least one router must be specified
       if (routers.isEmpty) throw NoRoutersSpecified
 
-      val telemeters = telemetry match {
-        case None => Seq(defaultTelemeter)
-        case Some(telemeters) => telemeters.map {
-          case t if t.disabled =>
-            val msg = s"The ${t.getClass.getCanonicalName} telemeter is experimental and must be " +
-              "explicitly enabled by setting the `experimental' parameter to `true'."
-            throw new IllegalArgumentException(msg) with NoStackTrace
-          case t => t.mk(Stack.Params.empty + param.LinkerConfig(this))
-        }
+      val metrics = MetricsTree()
+
+      val telemeterConfigs = telemetry match {
+        case Some(telemeters) if telemeters.exists(_.kind == "io.l5d.commonMetrics") =>
+          telemeters
+        case Some(telemeters) =>
+          telemeters :+ CommonMetricsConfig()
+        case None =>
+          Seq(CommonMetricsConfig())
```

Comment:
I'm not clear on this.  We always add CommonMetricsConfig?  This doesn't use MetricsTree, though, right? Are jvm stats, etc exported to _both_ common metrics and metrics tree?  What happens if both this and AdminTelemeter are configured?